components/InputReleaseAbout/InputReleaseAbout.jsx

```diff
@@ -0,0 +1,39 @@
+import dynamic from "next/dynamic"
+// import Quill from "react-quill"
+import "react-quill/dist/quill.snow.css"
+
+const QuillNoSSRWrapper = dynamic(import("react-quill"), {
+  ssr: false,
+  loading: () => <p>Loading...</p>,
+})
+
+const modules = {
+  toolbar: [
+    [{ header: "2" }],
```

Comment:
Remove this. I don't think folks need the ability to add headers if this is just a blurb.

Reply:
Removed, and also added alignment options.